snippet-extractor-output/snippets.yaml

```diff
@@ -3312,6 +3312,26 @@
 
         await context.sync();
        });
+'Excel.Error:class':
```

Comment:
Actually, I think this needs to map to [CustomFunctions.Error](https://docs.microsoft.com/en-us/javascript/api/custom-functions-runtime/customfunctions.error?view=excel-js-preview)

Reply:
Removing mapping per offline conversation.